pulumi/infra/config.py

```diff
@@ -213,18 +213,26 @@ def require_artifact(artifact_name: str) -> Any:
     return artifact
 
 
-def container_repository() -> Optional[str]:
-    """The repository from which to pull container images from.
+def cloudsmith_repository_name() -> Optional[str]:
+    """The repository from which to pull container images and Firecracker
+    packages from.
 
     This will be different for different stacks; we promote packages
     through a series of different registries that mirrors the progress
     of code through our pipelines.
 
-    The value will be something like
-    `docker.cloudsmith.io/grapl/testing`; to target a specific image
-    in client code, you would append the image name and tag to the
-    return value of this function.
+    The value will be something like `grapl/testing`.
+    """
+    return pulumi.Config().get("cloudsmith-repository-name")
+
+
+def container_repository() -> Optional[str]:
+    """The repository from which to pull container images from.
 
     Not specifying a repository will result in local images being used.
```

Comment:
"... but only for 'local grapl' deployments"

Reply:
fixed